### DISCUSSION TITLE
What kind of activities are you interested in?

### DISCUSSION THREAD

**Discussion by samnang** [web3together/web3together#4] [Ideas]
We want to explore some activities to engage people come to learn and share with each other in the community:

* 🗣 Bi-Weekly or Monthly meetup
  * How to get started in the blockchain technology
  * New technology in the blockchain space
  * How to build products(DeFi, NFTs, DAO, ...) in the blockchain ecosystem
  * Best practices in blockchain development
* 🔦 Smart Contact Code Reading
* 📖 Book club
* 🛠 Hackathon

We would like to hear some feedback from our community members.

**Comment by invisal** [comment]
Should include drinking. It is productive.

**Reply by samnang** [reply]
Not sure, people nowadays prefer remote than meeting in person.

**Reply by CHOMNANP** [reply]
I prefer meeting in person :D

**Comment by BunHouth** [comment]
NFTs + Marketplace should be great. :)

**Reply by samnang** [reply]
NFTs are the hot topic. I hope we could explore on this topic in near future with our members.

Could you share us your experiences on this topic?

**Reply by BunHouth** [reply]
I don't have much experience on it yet, due just got some work about it(NFTs Platform + Marketplace)

**Reply by samnang** [reply]
Oh, that's great. It will be helpful to get insights from you when we explore the topic.

Could you help to introduce yourself [here](https://github.com/web3together/web3together/discussions/2)? So other members can know you more.

**Reply by BunHouth** [reply]
sure, done.

**Comment by borinmin** [comment]
Great that you proposed this idea. I just have heard about block-chained but never deeply understood and never know how good it is. I would love to contribute if you plan to build the application using block-chained technology.

**Reply by samnang** [reply]
I'm glad you are interested in the topic. It would be nice if you could introduce [yourself](https://github.com/web3together/web3together/discussions/2), so other members can know each other.

**Comment by khousuylong** [comment]
Count me it

**Reply by samnang** [reply]
I'm so happy to see our friends are interested in the topic. It has been awhile since we ran the community together.

Could you introduce [yourself](https://github.com/web3together/web3together/discussions/2) to the members?

**Comment by sreylaut** [comment]
no experience in blockchain development but interesting.

**Reply by samnang** [reply]
Newbies are always welcome. I hope you will be something useful in this community. Feel free to ask anything questions or giving any feedbacks.
